public/components/agents/vuls/inventory/detail.tsx

```diff
@@ -0,0 +1,381 @@
+                          aria-label="Next"
+                          iconSize="s"
+                          className="buttonAddFilter"
+                        />
+                      </EuiToolTip>
+                    )}
+                  </span>
+                )
+              }
+              description={
+                <span>
+                  {item.icon !== 'users' ? (
+                    <EuiIcon size="l" type={item.icon} color="primary" className="detail-icon" />
+                  ) : (
+                    this.userSvg
+                  )}
+                  <span className="detail-title">{item.name}</span>
+                </span>
+              }
+              textAlign="left"
+              titleSize="xs"
+            />
+          </EuiFlexItem>
+        );
+      }
+    });
+
+    return (
+      <div>
+        <EuiFlexGrid columns={3}> {generalDetails} </EuiFlexGrid>
+      </div>
+    );
+  }
+
+  updateTotalHits = (total) => {
+    this.setState({ totalHits: total });
+  };
+
+  renderDetailsPermissions(value) {
+    if (((this.props.agent || {}).os || {}).platform === 'windows' && value && value !== '-') {
+      const components = value
+        .split(', ')
+        .map((userNameAndPermissionsFullString) => {
+          const [_, username, userPermissionsString] = userNameAndPermissionsFullString.match(
+            /(\S+) \(allowed\): (\S+)/
+          );
+          const permissions = userPermissionsString.split('|').sort();
+          return { username, permissions };
+        })
+        .sort((a, b) => {
+          if (a.username > b.username) {
+            return 1;
+          } else if (a.username < b.username) {
+            return -1;
+          } else {
+            return 0;
+          }
+        })
+        .map(({ username, permissions }) => {
+          return (
+            <EuiToolTip
+              key={`permissions-windows-user-${username}`}
+              content={permissions.join(', ')}
+              title={`${username} permissions`}
+            >
+              <EuiBadge color="hollow" title={null} style={{ margin: '2px 2px' }}>
+                {username}
+              </EuiBadge>
+            </EuiToolTip>
+          );
+        });
+      return (
+        <TruncateHorizontalComponents
+          components={components}
+          labelButtonHideComponents={(count) => `+${count} users`}
+          buttonProps={{ size: 'xs' }}
+          componentsWidthPercentage={0.85}
+        />
+      );
+    }
+    return value;
+  }
+
+
+  render() {
+    const { type, implicitFilters, view, currentItem, agent } = this.props;
+    const id = `${currentItem.name}-${currentItem.cve}-${currentItem.architecture}-${currentItem.version}`;
+    const inspectButtonText = view === 'extern' ? 'Inspect in Dashboard' : 'Inspect in Events';
+    return (
+      <Fragment>
+        <EuiAccordion
+          id={id === undefined ? Math.random().toString() : `${id}_details`}
```

Comment:
I am not sure about this. When the `id` variable will be `undefined`? THe `EuiAccordion` needs an `id` prop?